src/main.cpp

```diff
@@ -544,8 +543,6 @@ int main(int argc, char* argv[])
 #endif
 
     mudlet::debugMode = false;
-    FontManager fm;
-    fm.addFonts();
```

Comment:
I *was* going to point out about how can we use the font loaded here for the text on the splashscreen - but then I saw that we do not use any of the Bitstream Vera fonts that this installs (if included in the package) but a Deja Vu one instead - so when we say we are "*loading the fonts*" on the splash-screen we are *not really doing that* just copying them (the first time) out of the `mudlet_fonts` resource file - but I guess that is as accurate as the later message that reports "*All data has been loaded successfully.*" - 😝

So, it is okay after all to move the instantiation of the `FontManager` singleton to the `mudlet` constructor initialiser list IMHO - but I would suggest the instance is put as near to the top as possible of that list so that any other GUI classes (that might be stylesheet overridden to use the Ubuntu or Bitstream fonts I suppose) has access to them when those classes are created...